src/lib/car.js

```diff
@@ -126,14 +129,16 @@ async function storeCar({ id, skipExists, logger }) {
       contentLength: source.stats.contentLength
     },
     indexing: {
-      // @todo - consider including duration as nanoseconds instead of startTime/endTime milliseconds-resolution Dates
       startTime: new Date(Number(start) / 1e6),
-      endTime: new Date()
+      endTime: new Date(),
```

Comment:
do we need `endTime` along with `duration`?

Reply:
@simone-sanfratello for now we do because for now [metrics-collector](https://github.com/elastic-ipfs/metrics-collector) will still use just `endTime` and `startTime` to calculate duration.

But once we merge/ship this, I can [do this plan](https://github.com/elastic-ipfs/metrics-collector/issues/33#issuecomment-1209995302)
* ship an update to metrics-collector to use `duration` when present
  * also test to make sure that events without `indexing.endTime` are processed as expected
* verify that no systems (e.g. metrics-collector staging/prod) rely on `indexing.endTime` to work, and that event-delivery-lambda queue has no events without that `duration`
* make a PR here to indexer-lambda that remove `indexing.endTime` property and ship it